deepcell_tracking/tracking_test.py

```diff
@@ -197,71 +231,73 @@ def test_track_cells(self):
                     if exc.errno != errno.ENOENT:  # no such file or directory
                         raise  # re-raise exception
 
-    def test_fetch_tracked_features(self):
-        length = 128
-        frames = 5
+    # def test_fetch_tracked_features(self):
+    #     length = 128
+    #     frames = 5
```

Comment:
Just remove the code - no reason to leave it here commented out, we have it in the last releases.

Remove the code from the `tracking.py` too.